Source/Wit/Private/Wit/Request/HTTP/WitHttpRequest.cpp

```diff
@@ -81,8 +82,8 @@ void FWitHttpRequest::Tick(float DeltaSeconds)
 
 /**
  * Get the user agent to use for Wit requests
- * If Voice SDK installed: voice-sdk-46.0.2,voicesdk-unreal,Windows-10.0.xx.64bit,Unknown,xxx,Unknown,Editor
- * If Voice SDK is NOT installed:  wit-unreal-46.0.2,wit-unreal,Windows-10.0.xx.64bit,Unknown,xx,Unknown,Editor
+ * If Voice SDK installed: voicesdk-unreal-49.0.1,wit-unreal-49.0.1,"Windows-10.0.19045.1.256.64bit","GenuineIntel|Intel(R)Core(TM)i7-10750HCPU@2.60GHz",C6C3C9D74AE98762558677BF627B5666,com.Meta.voicesdk_unreal_demo,Editor,4.27.2
```

Comment:
I think the version no here is wrong and needs to be updated.

Reply:
this is just an example in the comment, whenever we release another version, which would involve a separate PR, the version code in UserAgent will be updated at runtime.